core/src/test/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorterSuite.java

```diff
@@ -101,7 +101,7 @@ public OutputStream apply(OutputStream stream) {
   public void setUp() {
     MockitoAnnotations.initMocks(this);
     sparkConf = new SparkConf();
-    tempDir = new File(Utils.createTempDir$default$1());
+    tempDir = Utils.createTempDir(System.getProperty("java.io.tmpdir"), "unsafe-test");
```

Comment:
Ok, I'll do this. I'll fix this test this way, then file a separate bug to have a shared, Java createTempDir() and use it from everywhere. How does that sound?

This is the only unit test that consistently fails in our builds so I'd like to at least unblock it.